sql/derived_tables/po_lines_eresource.sql

```diff
@@ -18,12 +17,22 @@ WITH temp_eresource AS (
         json_extract_path_text(data, 'eresource', 'userLimit') AS pol_user_limit,
         json_extract_path_text(data, 'eresource', 'resourceUrl') AS pol_resource_url
     FROM
-        po_lines AS pol
+        po_lines AS pol WHERE json_extract_path(data, 'eresource') IS NOT NULL
 )
```

Comment:
**row 13**
===============
`json_extract_path_text(data, 'eresource', 'license') AS pol_license`
documentation
` "license": {
            "code": "Code 1",
            "description": "License Agreement 1"
          },`
The documentation refers to subfields here. On line 14, the description is accessed as a subfield. In line 13 everything. Do you mean just the subfiled "code" here?

**row 20**
===============
Please put the WHERE clause on a new line and then the condition on a new line with indentation.